### ISSUE TITLE
Leaf names

### ISSUE THREAD

**Issue by timregan** [YoshioNakano2021/phy5#1]
Hi there,
Very easy to use shiny, well done and many thanks!
I have a problem with the names of the bacterial strains as they appear on the resultant tree, however.
Each of the closely related bacteria strains I am analysing has 2 chromosomes, although some of the assemblies are made up of many contigs. I assume this is being handled correctly, but the names of the strains seems to be coming from the name of the first sequence in the associated .fasta (see image).

e.g. rather than "Vibrio aestuarianus subsp. cardii strain 15/075 3T2" 
I am seeing 
"JAAKZL010000100.1 Vibrio aestuarianus subsp. cardii strain 15/075 3T2 NODE_100_length_9778_cov_28.506683, whole genome shotgun sequence"

Could you please help with this?

![image](https://github.com/YoshioNakano2021/phy5/assets/16758174/c387336a-3c25-4dc7-8a7e-3378a5cb5bb4)

**Comment by YoshioNakano2021** [comment]
Hi timregan,

Thank you for having your interest on phy5.
It has been updated and now the words "whole genome shotgun sequence" is removed from a strain name in the tree.
In addition, if a strain name contains the character string, " NODE ," "NODE" and the strings after that are removed.
Please try and let me know the result.

**Comment by timregan** [comment]
Much improved, thank you for the speedy response!
However, I am still seeing "contig" after some of the strains.
It would also be great to have the accessions removed before the strain name. Do you think this is possible?

**Comment by YoshioNakano2021** [comment]
Yes, it is possible, but I am not sure the word "contig" is always unnecessary for everyone.  So I have decided to give a function of "contig"-removal to the application for a limited period, for several months. Now you can try the modified Phy5.
